docs: Update Installation doc (#2162)
kubeshop/testkube: docs/installing.md

# Installation Steps

To get Testkube up and running you need to:

> 1. Install the Testkube CLI

Reviewer: ```suggestion
1. Install the Testkube CLI.
```

2. Use HELM or the Testkube CLI to to install Testkube Server components in your cluster.
3. (optional) Configure Testkube's Dashboard UI Ingress for your ingress-controller, if needed.

Watch the full installation video from our product experts: [Testkube Installation Video](https://www.youtube.com/watch?v=bjQboi3Etys).

# **1. Testkube CLI**
Package dependencies:
- [Kubectl](https://kubernetes.io/docs/tasks/tools/)

## From Scripts
To install on Linux or MacOs, run
```sh
bash < <(curl -sSLf https://kubeshop.github.io/testkube/install.sh )
```
## Through Package Managers
### **Homebrew (MacOS)**

You can install Testkube from Homebrew:
```sh
brew install testkube
```

> Or directly from our tap. The Homebrew mantainers take a few days/or week to approve each one of our releases so you can use our tap to make sure you always have the most recent release.

Reviewer: ```suggestion
Or directly from our tap. The Homebrew mantainers take a few days or a week to approve each one of our releases so you can use our tap to make sure you always have the most recent release.
```

```sh
brew tap kubeshop/homebrew-testkube
brew install kubeshop/testkube/testkube
```

### **Chocolatey (Windows)**

> You can use [Chocolatey](https://chocolatey.org/install) to install  version from our own chocolatey repository .

Reviewer: ```suggestion
You can use [Chocolatey](https://chocolatey.org/install) to install a version from our own chocolatey repository .
```

```sh
choco source add --name=testkube_repo --source=http://chocolatey.testkube.io/chocolatey
choco install testkube
```

#### **APT (Debian/Ubuntu)**

> 1. Download our public GPG key, and add them to the trusted keys:

Reviewer: ```suggestion
1. Download our public GPG key, and add it to the trusted keys:
```

```sh
wget -qO - https://repo.testkube.io/key.pub | sudo apt-key add -
```
2. Add our repository to your apt sources:
```sh
echo "deb https://repo.testkube.io/linux linux main" | sudo tee -a /etc/apt/sources.list
```
3. Make sure to get the updates:
```sh
sudo apt-get update
```

4. Install Testkube:
```sh
sudo apt-get install -y testkube
```

## **Manual Download**

> If you don't want automatic scripts or package managers you can always do a manual install:

Reviewer: ```suggestion
If you don't want to use automatic scripts or package managers you can always do a manual install:
```

> 1. Download binary with the version of your choice and platform of your choice [here](https://github.com/kubeshop/testkube/releases)

Reviewer: ```suggestion
1. Download the binary with the version of your choice and platform of your choice [here](https://github.com/kubeshop/testkube/releases)
```

> 2. Unpack it. For example, (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz) for Linux

Reviewer: ```suggestion
2. Unpack it. For example,  in Linux use (tar -zxvf testkube_0.6.5_Linux_arm64.tar.gz).
```

3. Move it to a location in the PATH. For example, `mv  testkube_0.6.5_Linux_arm64/kubectl-testkube /usr/local/bin/kubectl-testkube`.

For Windows, you will need to unpack the binary and add it to `%PATH%` as well.

If you use a package manager that we don't support, please let us know here [#161](https://github.com/kubeshop/testkube/issues/161).

> # **2. Testkube Server Componets**

Reviewer: ```suggestion
# **2. Testkube Server Components**
```

To deploy Testkube to your K8s cluster you will need the following packages installed:
- [Kubectl docs](https://kubernetes.io/docs/tasks/tools/) 
- [Helm docs](https://helm.sh/docs/intro/install/#through-package-managers)

> ## Using Testkube's CLI to deploy the Server Components

Reviewer: ```suggestion
## Using Testkube's CLI to Deploy the Server Components
```

> The Testkube CLI provides an command to deploy the Testkube server components to your cluster easly.

Reviewer: ```suggestion
The Testkube CLI provides a command to easily deploy the Testkube server components to your cluster.
```

Run:
```shell
testkube init
```

> note: you must have your KUBECONFIG ponting to the desired location of the installation.

Reviewer: ```suggestion
Note: you must have your KUBECONFIG pointing to the desired location of the installation.
```

By default Testkube is installed in the `testkube` namespace.

> ## **Using HELM to deploy the Server Components**

Reviewer: ```suggestion
## **Using HELM to Deploy the Server Components**
```

>For more configuration parameters of `MongoDB` chart please visit:
<https://github.com/bitnami/charts/tree/master/bitnami/mongodb#parameters>

## **Remove Testkube Server Components**
Using helm
```sh

> helm delete testkube

Author: I think testkube is installed by default in the "testkube" namespace right ?
Should it be "-n testkube" ?
